x-pack/plugins/actions/README.md

```diff
@@ -99,7 +99,8 @@ Built-In-Actions are configured using the _xpack.actions_ namespace under _kiban
 
 #### **allowedHosts** configuration
 
-- You can use the string "*" in the **allowedHosts** configuration in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential to use such a feature to execute [SSRF](https://www.owasp.org/index.php/Server_Side_Request_Forgery) attacks from your server.
+- You can use the string "*" in the **allowedHosts** configuration in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential to use such a feature to create [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery) attacks from your server.
```

Comment:
This is warning about a bad actor using actions to attack another server - create feels wrong, as an attack is performed or launched 🤔

Reply:
Thanks, I changed it to "launch"

Reply:
🚀